packages/terra-site/src/examples/dynamic-grid/ResponsiveGrid.jsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import DynamicGrid from 'terra-dynamic-grid';
+
+import Card from './Card';
+
+const layouts = [{
+  'grid-template-columns': '100px 100px 100px',
+  'grid-template-rows': '100px 100px 100px',
+  'grid-gap': '10px',
+  regions: [
+    {
+      name: 'r11',
+      'grid-column-start': 1,
+      'grid-row-start': 1,
+    },
+    {
+      name: 'r12',
+      'grid-column-start': 2,
+      'grid-row-start': 1,
+    },
+    {
+      name: 'r13',
+      'grid-column-start': 3,
+      'grid-row-start': 1,
+    },
+    {
+      name: 'r21',
+      'grid-column-start': 1,
+      'grid-row-start': 2,
+    },
+    {
+      name: 'r22',
+      'grid-column-start': 2,
+      'grid-row-start': 2,
+    },
+    {
+      name: 'r23',
+      'grid-column-start': 3,
+      'grid-row-start': 2,
+    },
+    {
+      name: 'r31',
+      'grid-column-start': 1,
+      'grid-row-start': 3,
+    },
+    {
+      name: 'r32',
+      'grid-column-start': 2,
+      'grid-row-start': 3,
+    },
+    {
+      name: 'r33',
+      'grid-column-start': 3,
+      'grid-row-start': 3,
+    },
+    {
+      name: 'cell',
+      'grid-column-start': 1,
+      'grid-row-start': 1,
+    },
+  ],
+}, {
+  media: '@media (max-width: 1500px)',
```

Comment:
Are these media layouts overlaid on top of the base configuration specified above? I'm wondering If theres a way to reorganize the API to make that more clear. Maybe the base config has an array of Media to regions pairs? Maybe at that point we could break up the monolithic API too.